src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -22,5 +22,9 @@ internal static class DiagnosticEventConstants
 
         public const string UnableToParseHostConfigurationFileErrorCode = "AZFD0009";
         public const string UnableToParseHostConfigurationFileHelpLink = "https://go.microsoft.com/fwlink/?linkid=2248917";
+
+        // TODO: Update this link with the proper docs once they get published
```

Comment:
you probably don't need this //TODO in here, right? We just need to write the docs as soon as this goes in?

Reply:
yup removed!